src/test/java/io/github/mfvanek/pg/connection/HighAvailabilityPgConnectionClusterTest.java

```diff
@@ -93,4 +101,37 @@ void standbyBecomesPrimaryOnPrimaryDownWithDefaultDelay() {
                 .as("First connection is not primary")
                 .isNotEqualTo(firstConnection);
     }
+
+    @Test
+    @Order(3)
+    void noPrimaryUpdateWithinUpdatePeriod() {
+        startFirstContainer();
+        final PgConnection firstConnection = getFirstPgConnection();
+        final PgConnection secondConnection = getSecondPgConnection();
+
+        final ArrayList<PgConnection> pgConnections = new ArrayList<>();
```

Comment:
1. Use `List<PgConnection>` as a variable type
2. Use Arrays.asList() to create a List

Reply:
here and above in this class

Reply:
As usual, I'm in a hurry and miss such an obvious smell :)